### ISSUE TITLE
Profession tools not sorted into profession bags

### ISSUE THREAD

**Issue by chaosblad3** [shirsig/SortBags#17]
Profession bags are capable of storing the tools used for that profession, e.g. mining pick can go in mining bag, jeweler's kit in gem pouch etc, however when sorting bags these tools are always moved into a general bag instead of their appropriate profession bag.

**Comment by shirsig** [comment]
yeah, I will try to finish it this weekend it's a lot of boring copy pasting of ids from wowhead.

**Comment by siweia** [comment]
The sortbags is not functioning, if you have a item with a socking gem in your container.

Change 'item:(%d+):(%d*):::::(%-?%d*):(%-?%d*)' into 'item:(%d+):(%d*):%d*::::(%-?%d*):(%-?%d*)' would fix the issue.
Turns out the gemID is next to enchantID.

Sorry that I post here.

**Comment by shirsig** [comment]
thanks, good catch. I've uploaded a fix

**Comment by Mayron** [comment]
I was about to say the same but the latest update fixed it for my leatherworking bag. Thanks for maintaining this :)

Also, the README says it's for "Classic WoW (1.13)" but I've been using it successfully for BC classic 20501 so maybe the README needs updating? Thanks again.
